hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerProtocolServerSideTranslatorPB.java

```diff
@@ -219,7 +244,7 @@ private OMResponse submitRequestDirectlyToOM(OMRequest request) {
         omClientResponse = omClientRequest.validateAndUpdateCache(
             ozoneManager, index, ozoneManagerDoubleBuffer::add);
       }
-    } catch(IOException ex) {
+    } catch(OMException ex) {
```

Comment:
I think we don't need this catch block at all, as the caller will do the right thing via the call to `createErrorResponse`.

Reply:
Addressed comments in
https://github.com/apache/hadoop-ozone/pull/12